docs/kqueen.rst

```diff
@@ -52,7 +52,9 @@ Serializers
 Configuration
 --------------
 
-Example configuration files are located in `config/` directory. Default configuration file is `config/dev.py` and this can be configured by `KQUEEN_CONFIG_FILE` environment variable. Any environment variable matching name `KQUEEN_*` will be loaded and merged with configuration file.
+Sample configuration files are located in the `config/` directory. The default configuration file is `config/dev.py`.
+To define a different configuration file set `KQUEEN_CONFIG_FILE` environment variable.  To override the values defined
```

Comment:
To define a different configuration file, set the `KQUEEN_CONFIG_FILE` environment variable.